src/navigations/root-stack.js

```diff
@@ -57,6 +57,7 @@ import {InitialStartupAtom, LoadingStartupContext} from '../service/initialStart
 import {NavigationConstants} from '../utils/constants';
 import {followersOrFollowingAtom} from '../screens/ChannelListScreen/model/followersOrFollowingAtom';
 import {useInitialStartup} from '../hooks/useInitialStartup';
+import {COLORS} from '../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `COLORS` from `../utils/theme` has been added. This is necessary for the changes in color scheme that have been made throughout the application.

```diff
+ import {COLORS} from '../utils/theme';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->